modules/navier_stokes/src/variables/INSFVVelocityVariable.C

```diff
@@ -65,6 +65,41 @@ INSFVVelocityVariable::INSFVVelocityVariable(const InputParameters & params) : I
 }
 
 #ifdef MOOSE_GLOBAL_AD_INDEXING
+
+const ADReal &
+INSFVVelocityVariable::getExtrapolatedBoundaryFaceValue(const FaceInfo & fi) const
+{
+  if (_two_term_boundary_expansion)
+    if (isFullyDevelopedFlowFace(fi))
```

Comment:
Can just group these two ifs together and reduce indentation

Reply:
The reason why I thought about doing this is that it is easy to check `_two_term_bc==true` but the `FullyFevelopedFlowFace(fi)` requires a warehouse lookup every time it is called. I am not sure if the lookups are cached or how expensive they can be, so this was a micro-optimization from my side. Let me know if you think this has close to zero gain and I'll  merge the two conditions.

Reply:
You know that c++ short-circuits right? For `if (A && B)`, if `A` is `false`, then `B` is not evaluated. So I don't see any gains here by having two separate ifs

Reply:
I did not know that, I'll merge the two then. Thanks for the new information!